packages/kit/src/runtime/server/index.js

```diff
@@ -94,7 +94,7 @@ export async function respond(request, options, state = {}) {
 		rawBody: body_getter
 	});
 
-	/** @type {import('types').RequiredResolveOptions} */
+	/** @type {Required<import('types').ResolveOptions>} */
```

Comment:
since this is used in a bunch of places perhaps it would make sense to have a `RequiredResolveOptions` type in `internal.d.ts`

Reply:
I thought about it, but it only saves two characters, so is it even worth it?